trieste/acquisition/rule.py

```diff
@@ -245,7 +262,7 @@ def __init__(
             builder = ExpectedImprovement().using(OBJECTIVE)
 
         if optimizer is None:
-            optimizer = optimize
+            optimizer = optimize_continuous
```

Comment:
does trust region not work with batches? Or perhaps this is something for another PR?